hyperparams/her.yml

```diff
@@ -92,6 +92,97 @@ FetchReach-v1:
   online_sampling: True
   normalize: True
 
+# Panda Robotic Env
```

Comment:
Please only update the TQC/SAC params as HER is now a replay bufffer and the hyperparams here are only there for legacy reasons.